sys-apps/musl-locales/musl-locales-0.1.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Locale program for musl libc"
+HOMEPAGE="https://git.adelielinux.org/adelie/musl-locales"
+SRC_URI="https://git.adelielinux.org/adelie/musl-locales/uploads/7e855b894b18ca4bf4ecb11b5bcbc4c1/${P}.tar.xz"
+
+LICENSE="LGPL-3"
```

Comment:
Do we need MIT here for the locales themselves?

Reply:
Fixed.